db/dbformat.h

```diff
@@ -44,7 +44,8 @@ static const size_t kL0_StopWritesTrigger = 12;
 // expensive manifest file operations.  We do not push all the way to
 // the largest level since that can generate a lot of wasted disk
 // space if the same key space is being repeatedly overwritten.
-static const int kMaxMemCompactLevel = 2;
+// Basho: push to kNumOverlapLevels +1 ... beyond "landing level"
+static const int kMaxMemCompactLevel = 3;
```

Comment:
changed as recommended.

> On Aug 8, 2015, at 4:09 PM, Paul A. Place <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> In db/dbformat.h <https://github.com/basho/leveldb/pull/163#discussion_r36581315>:
> 
> > @@ -44,7 +44,8 @@ static const size_t kL0_StopWritesTrigger = 12;
> >  // expensive manifest file operations.  We do not push all the way to
> >  // the largest level since that can generate a lot of wasted disk
> >  // space if the same key space is being repeatedly overwritten.
> > -static const int kMaxMemCompactLevel = 2;
> > +// Basho: push to kNumOverlapLevels +1 ... beyond "landing level"
> > +static const int kMaxMemCompactLevel = 3;
> The comment states that kMaxMemCompactLevel is kNumOverlapLevels +1, so why not just actually define it to be that value?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/basho/leveldb/pull/163/files#r36581315>.
>